### PR TITLE
helm: Add support of annotations in hubble ui service

### PR DESCRIPTION
**Pull request by brnck**
Signed-off-by: brnck <a.berneckas@gmail.com>

Please ensure your pull request adheres to the following guidelines:

- [X] For first time contributors, read [Submitting a pull request](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#submitting-a-pull-request)
- [X] All code is covered by unit and/or runtime tests where feasible.
- [X] All commits contain a well written commit description including a title,
      description and a `Fixes: #XXX` line if the commit addresses a particular
      GitHub issue.
- [ ] If your commit description contains a `Fixes: <commit-id>` tag, then
      please add the commit author[s] as reviewer[s] to this issue.
- [X] All commits are signed off. See the section [Developer’s Certificate of Origin](https://docs.cilium.io/en/stable/contributing/development/contributing_guide/#dev-coo)
- [X] Provide a title or release-note blurb suitable for the release notes.
- [X] Thanks for contributing!

We are using Cilium in EKS alongside with ALB ingress controller which requires the service to be annotated with particular information such as `alb.ingress.kubernetes.io/target-type: ip` in order to create a load balancer in the AWS. The problem is that the helm chart does not support annotating the service, therefore, we either need to manually annotate the service or use another hackish solutions 

```release-note
helm: Add support of annotations in hubble ui service
```

### CODE REVIEW

File: install/kubernetes/cilium/values.yaml [hunk 1]
```diff
@@ -1326,6 +1326,8 @@ hubble:
 
     # -- hubble-ui service configuration.
     service:
+      # -- Annotations to be added for the Hubble UI service
+      annotations: {}
```
Comment (kaworu) [inline install/kubernetes/cilium/values.yaml:1336]: Please report this change to `install/kubernetes/cilium/values.yaml.tmpl` and then run:
```
make -C install/kubernetes
```

**Review by kaworu — CHANGES_REQUESTED** [review]
Hi @brnck :wave: and thank you for the PR! Please see my comment, and also to fix the [travis-ci build](https://app.travis-ci.com/github/cilium/cilium/builds/260537315) you need to run:
```
make -C Documentation update-helm-values
```

**Review by sayboras — APPROVED** [review]
Apart from above point mentioned by Alex, the changes make sense and look good to me. Thanks.

**Comment by sayboras** [comment]
Also, can you run the below command to update helm docs as well

```bash
make -C Documentation update-helm-values
```

**Comment by brnck** [comment]
Hi, @kaworu thanks for the review! Executed both commands and committed the changes. Should be good now. 

Also, is it possible to backport this change to **1.12**?

**Review by kaworu — CHANGES_REQUESTED** [review]
> Hi, @kaworu thanks for the review! Executed both commands and committed the changes. Should be good now.

Thanks, the patch LGTM now, could you please sqash the three resulting commits into one?

> Also, is it possible to backport this change to 1.12?

Sure, adding the labels for v1.12 and v1.13 backport.

**Comment by brnck** [comment]
@kaworu Looks like I completely messed up to branch with squash :/ I think I'll close this PR and open a new one

**Pull request closed without merging**